package/MDAnalysis/core/AtomGroup.py

```diff
@@ -671,6 +749,7 @@ def force(self, vals):
         except (AttributeError, NoDataError):
             raise NoDataError("Timestep does not contain forces")
 
+    @deprecate(message="{}; use `position` property instead".format(_FIFTEEN_DEPRECATION))
     def centroid(self):
```

Comment:
This is for an ``Atom``, for which ``centroid`` is just its position. Unless there's a reason to keep this, I think it's unnecessary at best, confusing at worst. Happy to hear dissenting opinions, though.